File: 프로그래머스/0/120922. 종이 자르기/종이 자르기.js
function solution(M, N) {
    var answer = 0;
    var m = M
    var n = N
    var a = 0
    var b = 0
    
    while(m != 1) {
        m -= 1
        a += 1
    }
    
    while(n != 1) {
        n -= 1
        b += 1
    }
    
    answer = M * b + a
    
    return answer;
}

File: 프로그래머스/0/120922. 종이 자르기/종이 자르기.py
def solution(M, N):
    answer = 0
    m = M
    n = N
    a = 0
    b = 0
    
    while(m != 1):
        m -= 1
        a += 1
    print("a", a)
    
    while(n != 1):
        n -= 1
        b += 1
    
    print("b", b)
    print(M)
    
    answer = b * M + a
        
    return answer